system_api/cypher.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<p>[dust] I think you can remove cell output before saving a notebook -- removing this would make it look cleaner</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SystemInternal/SystemCookbook/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>